app/views/entities/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<div id='form'>
+  <h2>New entity</h2>
+
+  <%= render "form", entity: @entity %>
+
+  <br>
+
+  <div>
+    <%= link_to "Back to transactions", group_path(@group) %>
+  </div>
+</div>
```

Comment:
- [ ] Great job but part of the requirements is to follow the given design guidelines including:

- Colors
- Typography: font face, size and weight.
- Layout: composition and space between elements.

Kindly ensure you use the given font family `Proxima Nova` instead of `Roboto`